packages/ihe-gateway-sdk/src/models/shared.ts

```diff
@@ -77,6 +77,8 @@ export const baseResponseSchema = z.object({
   id: z.string(),
   timestamp: z.string(),
   responseTimestamp: z.string(),
+  requestTimestamp: z.string().optional(),
+  duration: z.number().optional(),
```

Comment:
We're not returning the duration from IHE GW, right? Left-over?

Reply:
No but were passing the duration to the `createOutboundPatientDiscoveryResp` which uses this schema

Reply:
So it seems its a more specialized prop? We shouldn't add props on the base schema if they're not used throughout, right?

Reply:
It will be used by PD, DQ and DR - unless it would be better to create another schema off of the base specifically to add the duration